python/cudf/cudf/tests/test_serialize.py

```diff
@@ -235,6 +235,21 @@ def test_serialize_all_null_string():
     assert_eq(recreated, pd_series)
 
 
+def test_serialize_named_series():
+    gdf = cudf.DataFrame({"a": [1, 2, 3, 4], "b": [5, 1, 2, 5]})
+
+    # Test basic named-series round trip
+    ser = gdf["b"]
+    recreated = cudf.Series.deserialize(*ser.serialize())
+    assert_eq(recreated, ser)
+
+    # Test seriesgroupby
```

Comment:
Lets split this into a separate test.

Reply:
Sounds good - done.

Reply:
Looks like this is still in the same test?